autofaiss/external/quantize.py

```diff
@@ -409,6 +411,7 @@ def score_index(
 def main():
     """Main entry point"""
     logging.basicConfig(level=logging.INFO)
+    logger.addHandler(logging.StreamHandler(sys.stdout))
```

Comment:
It is not necessary. Actually, I have tested it, I found that it would print twice for each log. You could simply run `python examples/memory_mapped_autofaiss.py` to reproduce.